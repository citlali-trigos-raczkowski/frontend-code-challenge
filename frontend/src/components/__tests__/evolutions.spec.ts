import { shallowMount, createLocalVue } from "@vue/test-utils";
import Evolutions from "../evolutions.vue";
import { sampleEvolutionPokemon } from "../../../tests/mocks/mock-props";
import VueRouter from "vue-router";

const localVue = createLocalVue();
localVue.use(VueRouter);

describe("component/evolutions.vue", () => {
  it("renders correctly", () => {
    const wrapper = shallowMount(Evolutions, {
      propsData: { pokemonEvolutions: [sampleEvolutionPokemon] },
      localVue,
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
