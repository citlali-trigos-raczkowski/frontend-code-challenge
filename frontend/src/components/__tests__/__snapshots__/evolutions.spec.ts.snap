// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component/evolutions.vue renders correctly 1`] = `
<div
  id="evolutions"
>
  <h2
    id="fall-left"
  >
    Evolutions
  </h2>
   
  <div
    id="evolution-container"
  >
    <pokemon-evolution-tile-stub
      pokemon="[object Object]"
    />
  </div>
</div>
`;
